AngularApp/README.md

```diff
@@ -11,8 +11,8 @@ Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app w
 Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
 
 ## Build
-
-Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `-prod` flag for a production build.
+1. Install npm packages by running `npm install`
```

Comment:
We can add this if you want but running npm install is sort of a given, and doesn't need to be done every time you build, which could be misleading.